test/AssociationRegistry.Invitations.Api.Tests/BijHetOpvragenVanUitnodigingen/OpVCode/GegevenEenAanvaarding.cs

```diff
@@ -47,7 +47,7 @@ public async Task DanBevatDeBodyDeGeregistreerdeUitnodiging()
         uitnodiging["uitnodiger"]["vertegenwoordigerId"].Value<int>().Should()
             .Be(_setup.Uitnodiging.Uitnodiger.VertegenwoordigerId);
         uitnodiging["uitgenodigde"]["insz"].Value<string>().Should().Be(_setup.Uitnodiging.Uitgenodigde.Insz);
-        uitnodiging["uitgenodigde"]["naam"].Value<string>().Should().Be(_setup.Uitnodiging.Uitgenodigde.Naam);
+        uitnodiging["uitgenodigde"]["naam"].Value<string>().Should().Be(_setup.Uitnodiging.Uitgenodigde.Achternaam);
```

Comment:
I believe uitnodiging["uitgenodigde"]["naam"] should become uitnodiging["uitgenodigde"]["achternaam"]